erpnext/docs/user/manual/en/accounts/purchase-invoice.md

```diff
@@ -3,9 +3,9 @@ that your Supplier sends you for products or services delivered. Here you
 accrue expenses to your Supplier. Making a Purchase Invoice is very similar to
 making a Purchase Order.
 
-To make a new Purchase Invoice, go to:
-
-> Accounts > Documents > Purchase Invoice > New Purchase Invoice
```

Comment:
@tundebabzy, I think we should also keep the breadcrumbs information

`Accounts > Documents > Purchase Invoice > New Purchase Invoice`